pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -93,14 +93,16 @@ def cache_logged_metrics(self) -> Union[EpochResultStore, None]:
         if self._current_stage is not None:
             self._cached_results[self._current_stage].cache_result()
 
-    def on_trainer_init(self, logger, flush_logs_every_n_steps, log_every_n_steps):
+    def on_trainer_init(self, logger, flush_logs_every_n_steps, log_every_n_steps, move_metrics_to_cpu):
```

Comment:
```suggestion
    def on_trainer_init(self, logger, flush_logs_every_n_steps: int, log_every_n_steps: int, move_metrics_to_cpu: bool):
```
also could we add simple docs?